components/ui/atoms/list-card-item/CardDescription.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "./CardItem.module.css";
+import { useCardProvider } from "./context/cardItemProvider";
+
+const CardDescription = () => {
+  const cardData = useCardProvider();
+  return <div className={styles.cardDescription}>{cardData?.description}</div>;
```

Comment:
```suggestion
  return cardData&&<div className={styles.cardDescription}>{cardData.description}</div>;
```
이런식으로 작성하는것도 방법일 수 있어용~